app/scripts/services/navigate.js

```diff
@@ -258,6 +258,11 @@ angular.module("openshiftConsole")
               group: rgv.group
             });
         }
+
+        if (_.get(opts, "tab")) {
+          url.removeSearch("tab").addSearch("tab", opts.tab);
```

Comment:
Had trouble with just using ```.setSearch``` overriding all search parameters, we'll double check though